hypothesis-python/tests/conjecture/test_shrinker.py

```diff
@@ -518,3 +519,33 @@ def shrinker(data: ConjectureData):
     # shrinking. Since the second draw is forced, this isn't possible to shrink
     # with just this pass.
     assert shrinker.choices == (15, 10)
+
+
+@pytest.mark.parametrize("start,boundary", [(-63, -46)])
+def test_shrink_small_integer_down_to_boundary(start, boundary):
+    @shrinking_from(ir(start))
+    def shrinker(data: ConjectureData):
+        n = data.draw_integer()
+        if boundary == 0:
+            data.mark_interesting()
+        elif boundary < 0:
+            if n <= boundary:
+                data.mark_interesting()
+        elif n >= boundary:
+            data.mark_interesting()
+
+    shrinker.fixate_shrink_passes(["minimize_individual_nodes"])
+    assert shrinker.shrink_target.ir_nodes[0].value == boundary
```

Comment:
`shrink_target.choices[0]` is a nice concise alternative here! I envision `.buffer[i]` -> `.choices[i]` being the default migration path for bytestring tests, though of course with tweaked indices.